pytest_avoidance.py

```diff
@@ -10,9 +10,12 @@
 import errno
 import sys
 import os
+import re
 
 CACHEROOT = None
 
+PARAMETERS_RE = re.compile(r'\[([^]]+)\]')
```

Comment:
This could/should be anchored at the end probably (`$`).

Reply:
Can you come up with a use case where that would make a difference?

Reply:
In theory you could have a test `test_foo[` that gets parametrized as `test_foo[[param1]`.
It won't make much a difference though I guess.

FWIW: I'm not using pytest-avoidance myself, just gave it a quick spin today.

Reply:
If you can show an actual test case where this breaks I'll be all ears :).

Here's inspiration:
https://github.com/walles/pytest-avoidance/blob/75fbc9ded38f1d881f4f67eec25390c291247966/tests/avoidance_test.py#L29-L37